src/renderers/dom/shared/__tests__/ReactBrowserEventEmitter-test.js

```diff
@@ -376,43 +374,30 @@ describe('ReactBrowserEventEmitter', () => {
   });
 
   it('should listen to events only once', () => {
-    spyOn(EventListener, 'listen');
+    spyOn(document, 'addEventListener');
     ReactBrowserEventEmitter.listenTo(ON_CLICK_KEY, document);
     ReactBrowserEventEmitter.listenTo(ON_CLICK_KEY, document);
-    expect(EventListener.listen.calls.count()).toBe(1);
+    expect(document.addEventListener.calls.count()).toBe(1);
   });
 
   it('should work with event plugins without dependencies', () => {
-    spyOn(EventListener, 'listen');
+    spyOn(document, 'addEventListener');
 
     ReactBrowserEventEmitter.listenTo(ON_CLICK_KEY, document);
 
-    expect(EventListener.listen.calls.argsFor(0)[1]).toBe('click');
+    expect(document.addEventListener.calls.argsFor(0)[0]).toBe('click');
   });
 
   it('should work with event plugins with dependencies', () => {
-    spyOn(EventListener, 'listen');
-    spyOn(EventListener, 'capture');
+    spyOn(document, 'addEventListener');
 
     ReactBrowserEventEmitter.listenTo(ON_CHANGE_KEY, document);
 
-    var setEventListeners = [];
-    var listenCalls = EventListener.listen.calls.allArgs();
-    var captureCalls = EventListener.capture.calls.allArgs();
-    for (var i = 0; i < listenCalls.length; i++) {
-      setEventListeners.push(listenCalls[i][1]);
-    }
-    for (i = 0; i < captureCalls.length; i++) {
-      setEventListeners.push(captureCalls[i][1]);
-    }
-
     var module = EventPluginRegistry.registrationNameModules[ON_CHANGE_KEY];
     var dependencies = module.eventTypes.change.dependencies;
-    expect(setEventListeners.length).toEqual(dependencies.length);
-
-    for (i = 0; i < setEventListeners.length; i++) {
-      expect(dependencies.indexOf(setEventListeners[i])).toBeTruthy();
```

Comment:
I deleted this set of assertions. I'm not sure what they're supposed to be testing but they didn't work in the first place.

The intention was probably to match whether an array contains an item, but `toBeTruthy()` check passes for `-1`. And `-1` is exactly what we get on master because `dependencies` array looks like this:

```js
      [ 'topBlur',
        'topChange',
        'topClick',
        'topFocus',
        'topInput',
        'topKeyDown',
        'topKeyUp',
        'topSelectionChange' ]
```

but value being tested looks like

```js
      'blur'
```

Maybe there's some better way to test this but I figured just asserting the count is enough.